test/helpers/web3Helper.ts

```diff
@@ -0,0 +1,6 @@
+const Web3 = require('web3'); // import web3 v1.0 constructor
+
+export const getWeb3 = () => {
+  const myWeb3 = new Web3('http://localhost:8545');
```

Comment:
Is this ideal? Or do we still want to set this elsewhere and pass it in

Reply:
perhaps make it so that provider can still be inputted but defaults to localhost:8545, so that this can be used elsewhere when we do want a custom provider